inc/layers.php

```diff
@@ -869,16 +869,7 @@ function get_layer($post_id = false) {
         $excerpt = apply_filters('translate_text', $post->excerpt, odm_language_manager()->get_current_language());
 
         $in_category = get_the_terms( $post->ID, 'layer-category' );
-        if (($key = array_search('map-catalogue', $in_category)) !== false) {
-            unset($array[$key]);
-        }
-        foreach ($in_category as $key => $in_cat) {
-          if($in_cat->slug == "map-catalogue"){
-            unset($in_category[$key]);
-          }
-        }
-        $in_category = array_values($in_category);
-
+        
```

Comment:
On the map explorer, there are some warming messages as below and I just try to fix them.

Warning: array_search() expects parameter 2 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 872

Notice: Undefined variable: array in wp-content\themes\wp-odm_theme\inc\layers.php on line 873

Warning: Invalid argument supplied for foreach() in wp-content\themes\wp-odm_theme\inc\layers.php on line 875

Warning: array_values() expects parameter 1 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 880

Warning: array_search() expects parameter 2 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 872

Notice: Undefined variable: array in wp-content\themes\wp-odm_theme\inc\layers.php on line 873

Warning: Invalid argument supplied for foreach() in wp-content\themes\wp-odm_theme\inc\layers.php on line 875

Warning: array_values() expects parameter 1 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 880

Warning: array_search() expects parameter 2 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 872

Notice: Undefined variable: array in wp-content\themes\wp-odm_theme\inc\layers.php on line 873

Warning: Invalid argument supplied for foreach() in wp-content\themes\wp-odm_theme\inc\layers.php on line 875

Warning: array_values() expects parameter 1 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 880

Warning: array_search() expects parameter 2 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 872

Notice: Undefined variable: array in wp-content\themes\wp-odm_theme\inc\layers.php on line 873

Warning: Invalid argument supplied for foreach() in wp-content\themes\wp-odm_theme\inc\layers.php on line 875

Warning: array_values() expects parameter 1 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 880

Warning: array_search() expects parameter 2 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 872

Notice: Undefined variable: array in wp-content\themes\wp-odm_theme\inc\layers.php on line 873

Warning: Invalid argument supplied for foreach() in wp-content\themes\wp-odm_theme\inc\layers.php on line 875

Warning: array_values() expects parameter 1 to be array, boolean given in wp-content\themes\wp-odm_theme\inc\layers.php on line 880